### PR TITLE
Nested subpackages example for cfg commands

### PR DESCRIPTION
**Pull request by phanimarupaka**
@mortent 

This is an example package with nested subpackages. This will be used in the guides to illustrate behavior of cfg commands with nested subpackages.

### CODE REVIEW

**Review by prachirp — CHANGES_REQUESTED** [review]
I know it's meant to be used in a guide but it's hard to review this as a standalone PR. Some suggestions:
- Rename wordpress-in-a-box to nested-subpackage to give more context on the motivation for the example.
- Add a README.md with a short description similar to the [config management package examples](https://github.com/GoogleCloudPlatform/csp-config-management/tree/1.0.0/ci-app).
- Can also add a README.md under package-examples with one-line descriptions of all the examples, see [here](https://github.com/GoogleCloudPlatform/csp-config-management).

File: package-examples/wordpress-in-a-box/mysql/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  namespace: PROJECT_ID-yourspace # {"$kpt-set":"proj-namespace"}
```
Comment (mortent) [inline package-examples/wordpress-in-a-box/mysql/deployment.yaml:4]: Why are we not using the `default` namespace as the default here?

**Review by mortent — COMMENTED** [review]
I agree with the comments from @prachirp. In particular, in what context to we plan to use this example?

File: package-examples/wordpress-in-a-box/mysql/deployment.yaml (hunk 1 above)
Comment (phanimarupaka) [reply package-examples/wordpress-in-a-box/mysql/deployment.yaml:4]: It is an example where package publisher marks namespace as a required setter and consumer must set it before using the blueprints.

**Comment by phanimarupaka** [comment]
@mortent @prachirp Moving this to a different PR with different example along with guides https://github.com/GoogleContainerTools/kpt/pull/1022

**Pull request closed without merging**